frontend/src/app/modules/user-profile/user-profile-page/user-profile.component.ts

```diff
@@ -143,6 +143,7 @@ export class UserProfileComponent extends BaseComponent implements OnInit, OnDes
                     this.spinner.hide();
                     this.notificationService.info('Names successfully updated');
                     this.initUserNamesForm();
+                    window.location.reload();
```

Comment:
do we really need to reload the whole page or probably just `this.fetchCurrentUser()` or smth like that?

Reply:
> do we really need to reload the whole page or probably just `this.fetchCurrentUser()` or smth like that?

This function will not update our drop-down. You need to either use the EventService to which our drop-down is subscribed or update our page as I did

Reply:
Let's not reload page, please

Reply:
> Let's not reload page, please

ok. I'll fix it now